docs/WritingTests.asciidoc

```diff
@@ -2094,6 +2094,14 @@ from the example distribution for an example. You may append a parameter like
 `SCENARIO_DEFINITIONS_YAML=path/of/yaml` to the query parameters of the webhook
 to change the lookup path of this file.
 
+=== Run isotovideo directly in the CI runner
+It is also possible to avoid using openQA at all and run the backend
+`isotovideo` directly within the CI runner. This simplifies the setup as no
+openQA instance is needed but of course test results cannot be examined using
+a web interface as usual. Checkout the
+https://github.com/os-autoinst/os-autoinst-distri-example/blob/main/README.md#local-testing-and-ci-environment[README of the example test distribution]
+for more information.
```

Comment:
With https://github.com/os-autoinst/os-autoinst/pull/2503 how about pointing to the os-autoinst README instead? The above explanation text about the web interface is of course unique and still makes sense to have here

Reply:
Both, os-autoinst and openQA are basically just referring to the example distribution. So I'd avoid the extra hop.